src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -611,37 +611,14 @@ protected TokenValidator createTokenValidator() {
             conf.setDisablePkce(true);
         }
         opMetadataResolver.init();
-        if (loginQueryParamNameValuePairs != null && !loginQueryParamNameValuePairs.isEmpty()) {
```

Comment:
moved validation to the `LoginQueryParameter`  all parameters are now deemed to be valid.